font/src/lib.rs

```diff
@@ -267,3 +278,11 @@ pub trait Rasterize {
     /// Update the Rasterizer's DPI factor
     fn update_dpr(&mut self, device_pixel_ratio: f32);
 }
+
+// Only implemented for the FreeType rasterizer so far.
+/// Conceptually this extends the Rasterizer trait with Harfbuzz specific functionality.
```

Comment:
I think we can make this comment a little bit more concise.

```suggestion
/// Extend the Rasterizer trait with Harfbuzz specific functionality.
```

Though generally, this is currently not an extension to the rasterizer trait, so we either need to reword it or change the `HbFtExt` trait to require the `Rasterizer` trait to be implemented. I don't care either way, since the `Rasterizer` trait will always be implemented anyways and the `HbFtExt` should technically be independent.

But we should still make things clear. If you should decide to remove the extension part, the trait also needs to be renamed though.

Reply:
:sweat_smile: It appears I've commented on the wrong reply. My reply to the comment about `// Only implemented for the FreeType rasterizer so far.` also applies to this comment. I'm gonna resolve this comment in the interest of consolidating discussion.